packages/element/README.md

```diff
@@ -270,7 +270,9 @@ _Related_
 
 ### RawHTML
 
-Component used as equivalent of Fragment with unescaped HTML, in cases where it is desirable to render dangerous HTML without needing a wrapper element. To preserve additional props, a `div` wrapper _will_ be created if any props aside from `children` are passed.
+Component used to render unescaped HTML.
+
+Note: The `renderElement` serializer will remove the `div` wrapper unless non-children props are present; typically when preparing a block for saving.
```

Comment:
I'm not sure if that is clear enough. The way I understand it, there will be always a `div` wrapper. The only exception is when seralizing the block and there are no props provided, the content will be rendered as is.

Reply:
Maybe, we could also include an example with the result:


```js
const Component = () => <RawHTML><h3>Hello world</h3></RawHTML>;
// Edit: <div><h3>Hello world</h3></div>
// save: <h3>Hello world</h3>
```

Reply:
Yeah, tried my best not to sound too technical, and this is the best I could come up with 😅 

I'll include code example 👍